### ISSUE TITLE
Will Sledge ever be portable?

### ISSUE THREAD

**Issue by Pyroglyph** [LogicAndTrick/sledge#219]
I know that Sledge is being developed and important features are yet to be implemented (such as Source support) but do you plan to make a portable version (as in one that doesn't make registry entries and also doesn't require admin privileges to install/run) because I would very much like to run it at my college to make a Garry's Mod map (the campus would make a fantastic map) and since Steam won't run there I have no other option really (although I am looking into Jackhammer and Source SDK).

tl;dr - Will Sledge ever have a portable version that doesn't need admin privileges to run?

**Comment by LogicAndTrick** [comment]
This is a good idea, a proper portable version should be possible, at least for the Goldsource engine. (Source needs the SDK to be installed for a lot of stuff, which makes portable versions kinda useless)

Can you use the zip release from the [releases page](https://github.com/LogicAndTrick/sledge/releases)? The installer needs admin access to register file associations and so on, but I don't think the editor does if you just extract the files from the zip. It's not exactly portable because it uses the AppData directory, but it should run. I might be wrong though because I haven't tested that in a long time...

You'd have to re-setup the game configuration on every PC you use, or copy the `%AppData%/Sledge` directory and make sure the game files (FGDs, Wads, etc) are always on the same drive path.
